src/test/scala/nl.knaw.dans.easy.deposit/servlets/IntegrationSpec.scala

```diff
@@ -43,8 +43,8 @@ class IntegrationSpec extends TestSupportFixture with ServletFixture with Scalat
   private val app: EasyDepositApiApp = new EasyDepositApiApp(minimalAppConfig) {
     override val pidRequester: PidRequester = mockPidRequester
 
-    override def getUser(user: String): Try[Map[String, Seq[String]]] = {
-      Success(Map(("displayName", Seq("F. Bar"))))
+    override def getUserProperties(user: String): Try[Map[String, Seq[String]]] = {
```

Comment:
As you do this in both `DepositServletFixture` and here, it begs the question whether you can extract something here... Perhaps `EasyDepositApiAppFixture` (or just `AppFixture`)?